admin_manual/configuration_server/occ_command.rst

```diff
@@ -1527,3 +1527,13 @@ Use the following command to enable the user again::
  sudo -u www-data php occ user:enable <username>
 
 Note that once users are disabled, their connected browsers will be disconnected.
+
+Run Reports
+-----------
+
+The ability to run reports by using the occ command, is available for ownCloud 9.1 and higher.
```

Comment:
On the stable9 branch of configreport it says 9.0: https://github.com/owncloud/configreport/blob/stable9/appinfo/info.xml